src/app/market/listings/preview-listing/preview-listing.component.html

```diff
@@ -75,12 +75,8 @@
       <strong>Date added</strong> {{ data?.listing?.createdAt }}
     </div>
   </div>
-  <div class="right" *ngIf="!data?.listing?.object.ListingItemTemplate?.hash">
-    <app-favorite [listing]="data?.listing"></app-favorite>
-
-    <button mat-raised-button color="primary" (click)="addToCart(data?.listing)">
-      <mat-icon fontSet="partIcon" fontIcon="part-check"></mat-icon>
-      Add to cart
-    </button>
+  <div class="right" fxLayout fxLayoutAlign="end center" *ngIf="!data?.listing?.object.ListingItemTemplate?.hash">
```

Comment:
Doesn't there exist a 'hash' object in the listing model?

try rewriting it as such:
*ngIf="!data?.listing?.hash"

Reply:
I thought we are checking the template related hash and guess both are same so we can make that the above :)